### PR TITLE
remove reference to es-grafana in Makefile

### PR DESCRIPTION
**Pull request by marcoderama**
PR checks are failing in Circle because the `make lint-json` target is failing.  This fixes that.  (The `es-grafana` directory was removed in PR https://github.com/lightbend/console-charts/pull/408 (aka a _long_ time ago...).

### CODE REVIEW

File: enterprise-suite/scripts/validate-promql.sh [hunk 1]
```diff
@@ -9,39 +9,39 @@ if ! command -v promtool > /dev/null; then
     exit 1
 fi
 
-if ! command -v jq > /dev/null; then
```
Comment (marcoderama) [inline enterprise-suite/scripts/validate-promql.sh:12]: Should consider moving all this stuff to the console-grafana repo now...

Comment (marcoderama) [reply enterprise-suite/scripts/validate-promql.sh:12]: Done in PR https://github.com/lightbend/console-grafana/pull/80

**Comment by JustinPihony** [comment]
Current failure:

> [1636008415] Ingress Suite - 1/1 specs 
> [It] responds to requests
>   /home/circleci/go/src/github.com/lightbend/console-charts/enterprise-suite/gotests/tests/ingress/ingress_test.go:44
> 10.128.2.145
> minikube ip: unable to copy process stdout: read |0: file already closedminikube ip: unable to copy process stderr: read |0: file already closed
> ------------------------------
> • Failure [70.424 seconds]
> minikube:ingress [It] responds to requests 
> /home/circleci/go/src/github.com/lightbend/console-charts/enterprise-suite/gotests/tests/ingress/ingress_test.go:44
> 
>   Expected success, but got an error:
>       <*errors.errorString | 0xc000239ad0>: {
>           s: "WaitUntilSuccess failed: console not accessible through ingress: Get \"http://10.128.2.145/es-console\": dial tcp 10.128.2.145:80: connect: connection refused",
>       }
>       WaitUntilSuccess failed: console not accessible through ingress: Get "http://10.128.2.145/es-console": dial tcp 10.128.2.145:80: connect: connection refused
> 
>   /home/circleci/go/src/github.com/lightbend/console-charts/enterprise-suite/gotests/tests/ingress/ingress_test.go:113
> ------------------------------
> 
> 
> Summarizing 1 Failure:
> 
> [Fail] minikube:ingress [It] responds to requests 
> /home/circleci/go/src/github.com/lightbend/console-charts/enterprise-suite/gotests/tests/ingress/ingress_test.go:113
> 
> Ran 1 of 1 Specs in 76.067 seconds

**Comment by marcoderama** [comment]
Done in PR https://github.com/lightbend/console-charts/pull/505.  Closing.

**Pull request closed without merging**